tests/permissions-read-only.spec.ts

```diff
@@ -166,20 +214,36 @@ test.describe("Given a user with Viewer Server permissions...", () => {
 
     // Bulk actions on instance list page
     await page
-      .locator("#instances-table span")
-      .filter({ hasText: "Select all" })
+      .getByRole("row", {
+        name: "select Name Type Description Status Actions",
+      })
+      .getByLabel("multiselect rows")
       .click();
+    await page.getByRole("button", { name: "Select all instances" }).click();
     await expect(
-      page.getByRole("button", { name: "Start", exact: true }),
+      page.getByTitle(
+        "You do not have permission to start the selected instances",
+      ),
```

Comment:
This fails, because we are only selecting one instance here. Maybe use a more robust way to select either only one instance or make sure the logic works with both one or many instances. We can encounter both depending on previous test runs, that can crash and leave behind instances.